clang/tools/scan-build-py/lib/libscanbuild/arguments.py

```diff
@@ -469,13 +469,13 @@ def parser_build_log(parser):
     parser.add_argument(
         '--parse-build-log',
         metavar='<file>',
-        help="""Specifies the build log file path of project specified by --in-root.""")
+        help="""Specifies the build log file path of project specified by --work-directory.""")
```

Comment:
how about use "Specifies the file path of the build log."

Reply:
OKay